server.js

```diff
@@ -342,7 +343,10 @@ app.use(CORS);
 
 if (listen && basicAuthMode) app.use(basicAuthMiddleware);
 
-app.use(whitelistMiddleware(enableWhitelist, listen));
+app.use(whitelistMiddleware(enableWhitelist));
+if (listen) {
+    app.use(accessLoggerMiddleware());
+}
```

Comment:
Do you prefer `listne && app.use(accessLoggerMiddleware());` or the current approach?